src/plugins/elasticsearch/index.js

```diff
@@ -19,7 +19,7 @@ module.exports = function ({ Plugin }) {
         username: string(),
         password: string(),
         shardTimeout: number().default(0),
-        requestTimeout: number().default(30000),
+        requestTimeout: number().default(300000),
```

Comment:
Huh, totally missed that. I think that's a typo in the config file, I think we should fix the value in the yml file and leave this value alone personally. The [default in the es client](https://www.elastic.co/guide/en/elasticsearch/client/javascript-api/current/configuration.html) is 30 seconds/30000.